PacketDotNet/TransportPacket.cs

```diff
@@ -20,6 +21,14 @@ namespace PacketDotNet
     /// </summary>
     public abstract class TransportPacket : Packet
     {
+        /// <summary>
+        /// Callback function for UdpPacket Payload decoding.
+        /// First parameter is the payload, second parameter if the UdpPacket itself
+        /// returned value is the decoded payload as Packet or null if payload is invalid or not supported
+        /// then internal decoding will continue
+        /// </summary>
```

Comment:
This comment should be updated to reflect transport packet.

Reply:
oh, sorry, yes of course.

Reply:
No problem 👍

Reply:
done